fortio_main.go

```diff
@@ -536,9 +537,23 @@ func grpcClient() {
 		}
 		return
 	}
-	_, err := fgrpc.PingClientCall(host, count, httpOpts.PayloadUTF8(), *pingDelayFlag, &httpOpts.TLSOptions)
+	_, err := fgrpc.PingClientCall(host, count, httpOpts.PayloadUTF8(), *pingDelayFlag, &httpOpts.TLSOptions, grpcMD())
 	if err != nil {
 		// already logged
 		os.Exit(1)
 	}
 }
+
+func grpcMD() map[string][]string {
+	md := make(map[string][]string)
+	headers := bincommon.SharedHTTPOptions().ExtraHeaders
+	for k, v := range headers {
+		k = strings.ToLower(k)
+		if k == "user-agent" {
+			// todo: map standard http headers to grpc md using the dial options
+			continue
+		}
+		md[k] = append(md[k], v...)
```

Comment:
what differences between that and
```
md[k] = v
```
? (copying instead of referencing maybe but is it necessary?)

Reply:
🤔I may have been a little careless, I'll fix it